README.md

```diff
@@ -1,54 +1,283 @@
+Playstation (PSX) | Octoshock ||
+PSP | PPSSPP† ||
+Sega Game Gear | SMSHawk ||
+Sega Genesis | Genplus-gx ||
+Sega Master System | SMSHawk ||
+Sega Saturn | Saturnus ||
+Sega Pico | PicoDrive ||
+SNES | BSNES/Snes9x | BSNES/Snes9x |
+TI-83 | TI83Hawk ||
+TurboGrafx / SuperGrafx | PCEHawk ||
+Uzebox | Uzem† ||
+Virtual Boy | Virtual Boyee ||
+WonderSwan / Color | Cygne ||
+ZX Spectrum | ZXHawk ||
+
+† core is in development and unreleased
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+## Support and troubleshooting
+
+A short [FAQ](http://tasvideos.org/Bizhawk/FAQ.html) is provided on the [BizHawk wiki](http://tasvideos.org/Bizhawk.html). If your problem is one of the many not answered there, and you can't find it in the [issue tracker search](https://github.com/TASVideos/BizHawk/issues?q=is%3Aissue+ISSUE_KEYWORDS), check the [BizHawk forum](http://tasvideos.org/forum/viewforum.php?f=64) at TASVideos, or ask on IRC:
+
+[![IRC | chat.freenode.net #bizhawk](https://img.shields.io/badge/IRC-chat.freenode.net_%23bizhawk-steelblue.svg?style=flat)](ircs://chat.freenode.net:6697/bizhawk)
+[![Matrix (IRC bridge) | #freenode_#bizhawk:matrix.org](https://img.shields.io/badge/Matrix_(IRC_bridge)-%23freenode__%23bizhawk:matrix.org-mediumpurple.svg?logo=matrix&style=flat)](https://matrix.to/#/#freenode_#bizhawk:matrix.org)
+[![freenode webchat | #bizhawk](https://img.shields.io/badge/freenode_webchat-%23bizhawk-lightcoral.svg?style=flat)](http://webchat.freenode.net/?channels=bizhawk)
+
+If there's no easy solution, what you've got is a bug. Or maybe a feature request. Either way, [open a new issue](https://github.com/TASVideos/BizHawk/issues/new) (you'll need a GitHub account, signup is very fast).
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+## Contributing
+
+BizHawk is Open Source Software, so you're free to modify it however you please, and if you do, we invite you to share! Under the MIT license, this is *optional*, just be careful with reusing cores as some have copyleft licenses.
+
+If you'd like to fix bugs, check the issue tracker here on GitHub:
+
+[![GitHub open issues counter](https://img.shields.io/github/issues-raw/TASVideos/BizHawk.svg?logo=github&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk/issues)
+
+It's a good idea to check if anyone is already working on an issue by asking on IRC (see *Support* above).
+
+If you'd like to add a feature, first search the issue tracker for it. If it's a new idea, make your own feature request issue before you start coding.
+
+For the time being, style is not enforced in PRs, only build success is. Please use CRLF, tabs, and [Allman braces](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) in new files.
+
+Past contrbutors to the frontend and custom-built cores are listed [here](https://github.com/TASVideos/BizHawk/graphs/contributors). See a core's docs for its authors.
+
+### Testing
```

Comment:
I believe this should go before Building.

Reply:
I don't know how to make that look good. Any ideas?

Reply:
I mean just move the chapter.

Reply:
done temporarily, take a look